src/System.Drawing.Primitives/ref/System.Drawing.Primitives.cs

```diff
@@ -358,7 +359,7 @@ public enum KnownColor
     }
     public partial struct Point : System.IEquatable<System.Drawing.Point>
     {
-        private int _dummy;
+        private int _dummyPrimitive;
```

Comment:
This types are serializable please don't rename fields.

/cc @ViktorHofer

Reply:
This is the ref assembly which isn't used for serialization. I don't have context why the field names are changed but it shouldn't cause an issue for serialization if it's just in the ref.

Reply:
That's just fallout from GenAPI changes made by @weshaggard / @ahsonkhan.

Reply:
this change was, i used /t:regenereate.. option to regenerate ref. I am leaving this change as-is.

Reply:
Yes please leave the change.